examples/stable_baselines3/README.md

```diff
@@ -0,0 +1,43 @@
+# SMARTS Stable Baselines3 Example
+This example illustrates the training of an ego agent to drive, as fast and as far as possible, in traffic using the Stable Baselines3 (https://github.com/DLR-RM/stable-baselines3) reinforcement-learning algorithms.
+
+Ego agent earns rewards based on the distance travelled and is penalised for colliding with other vehicles and for going off-road.
+
+## Trained agent driving in traffic
+![](./docs/_static/driving_in_traffic.gif)
+
+## Setup
+```bash
+$ git clone https://github.com/huawei-noah/SMARTS.git
+$ follow the SMARTS repository setup, including [camera-obs]
+$ cd <path>/SMARTS/examples/stable_baselines3
+$ pip install stable-baselines3
+$ scl scenario build-all --clean ./scenarios/loop
```

Comment:
Remove the need to build scenarios manually. Do automatically build the `scenarios` to make the example user friendly. See pr #1141 for example.